src/lib/components/Title.svelte

```diff
@@ -0,0 +1,15 @@
+<h4><slot /></h4>
```

Comment:
Probably `Title` is a bit a too generic name for so specific component but not a strong opinion.

Reply:
Not super happy with the name neither, better idea?

`HeaderTitle`?

Reply:
Done as discussed renamed to `HeaderTitle`